source/en-US/SqlServerDsc.strings.psd1

```diff
@@ -223,8 +223,8 @@ ConvertFrom-StringData @'
     ConvertTo_EditionName_ConvertingEditionId = Converting EditionId '{0}' to Edition name.
     ConvertTo_EditionName_UnknownEditionId = The EditionId '{0}' is unknown and could not be converted.
 
-    ## Assert-SqlLogin
-    AssertLogin_CheckingLogin = Checking if the principal '{0}' exists as a login on the instance '{1}'.
-    AssertLogin_LoginMissing = The principal '{0}' does not exist as a login on the instance '{1}'.
-    AssertLogin_LoginExists = The principal '{0}' exists as a login.
+    ## Assert-SqlDscLogin
+    AssertDscLogin_CheckingLogin = Checking if the principal '{0}' exists as a login on the instance '{1}'.
```

Comment:
Change the key name from AssertDscLogin_CheckingLogin to Assert_Login_CheckingLogin, same with the other two. Update the code accordingly.